artifacthub/designs/kubernetes_basic/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.0.0
+name: kubernetes_basic
+displayName: kubernetes_basic
+createdAt: "2024-03-21T14:19:01Z"
+description: Basic Kubernetes design deploying a service and pod
+# logoPath: ./../../logo.png Is there a logo to add? Could also use logoURL if it's hosted externally
+license: Apache-2 # Has there been discussion on default license for designs?
+homeURL: https://docs.meshery.io/concepts/logical/designs
+appVersion: 1.0.0
+keywords:
+    - kubernetes
+    - meshery
+    - service mesh
+    - orchestration
+links:
+    - name: website
+      url: https://meshery.io/
+    - name: support
+      url: https://discuss.layer5.io/
+install: |
+  Does Meshery need to be running in the cluster?
```

Comment:
What would it mean to install a design from AH? Would Meshery need to already be setup in the cluster? Or could someone install a design on a vanilla cluster?

Reply:
Great question, @ctcarrier. Here's a verbal explanation - 

https://github.com/meshery/schemas/assets/7570704/55de3a1b-ba7f-4168-8930-282d2fc13fcb

The first portion of the recording misses me saying, "Both Artifact Hub and Meshery Cloud Catalog function as central repositories for Meshery Designs...".

Generally, yes, the expectation is that an Artifact Hub user will be running their own Meshery Server, download the design from AH, then import into their Meshery Server.